packages/changed-elements-react/package.json

```diff
@@ -48,7 +48,6 @@
     "@itwin/core-frontend": "^4.0.6",
```

Comment:
Please add changelog entry.

Reply:
Pushed patch changes to unreleased section.